Project.toml

```diff
@@ -21,28 +24,29 @@ SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 StrideArrays = "d1fa6d79-ef01-42a6-86c9-f7c551f8593b"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
```

Comment:
Same here - can we manage to have just the ChainRulesCore dependency but not ChainRules and not Zygote? Note it is ok to have Lux and Zygote dependency in the tests.

Reply:
I think yes. I will do a clean up on it and use those two packages only in testing.